packages/ifpack2/src/Ifpack2_AdditiveSchwarz_def.hpp

```diff
@@ -550,6 +563,14 @@ apply (const Tpetra::MultiVector<scalar_type,local_ordinal_type,global_ordinal_t
            << "OverlappingMatrix_ is null when it shouldn't be.  "
            "Please report this bug to the Ifpack2 developers.");
         OverlappingMatrix_->exportMultiVector (*OverlappingY, *C, CombineMode_);
+
+        // average solution in overlap regions if requested via "schwarz: combine mode" "AVG"
+        if (AvgOverlap_) {
+          Teuchos::ArrayRCP<double>  dataC = C->getDataNonConst(0);
```

Comment:
It's weird that only the Zoltan2Sphynx [build](https://trilinos-cdash.sandia.gov/viewBuildError.php?buildid=396529) fails for SC=complex<double>.

Reply:
This is being tracked in https://github.com/trilinos/Trilinos/issues/10823#issuecomment-1206856350.